pkg/providers/v1/providers.go

```diff
@@ -143,6 +144,7 @@ type GitHub interface {
 	AddAuthToPushOptions(ctx context.Context, options *git.PushOptions) error
 	StartCheckRun(context.Context, string, string, *github.CreateCheckRunOptions) (*github.CheckRun, error)
 	UpdateCheckRun(context.Context, string, string, int64, *github.UpdateCheckRunOptions) (*github.CheckRun, error)
+	GetRepoLanguages(ctx context.Context, owner, repo string) (map[string]int, error)
```

Comment:
Do we need to expose this? This can probably jus the an implementation detail of how GitHub populates properties.